### ISSUE TITLE
Error saving comments

### ISSUE THREAD

**Issue by Ulyanov-programmer** [postcss/postcss-nested#162]
_Hello!_

While using your plugin, I noticed the following:

If attach a **comment** to a rule, regardless of whether it is embedded in another one or not, it will be displayed **correctly**. 
(**note that the `preserveEmpty` parameter is `true`**)
<details><summary>Example</summary>

```css
p {
  a {
    /* Text */
  }
}

/* to */

p {}

p a {
  /* Text */
}
```

</details> 

However, if a comment is nested in a rule that _has a different rule inside_, it will **not be nested**:

<details><summary>Example</summary>

```css
p {
  /* P Text */
  a {
    /* Text */
  }
}

/* to */

p {}

/* P Text */
p a {
  /* Text */
}
```

</details> 

But it was **expected**:

```css
p {
  /* P Text */
}

p a {
  /* Text */
}
```

**Comment by ai** [comment]
Position comments right in all cases is too hard for me. So if you want to fix it, you need to send PR.

**Comment by Ulyanov-programmer** [comment]
Done.
https://github.com/postcss/postcss-nested/pull/163

**Comment by ai** [comment]
Fixed in 7.0.
